src/drivers/mct/main/cime_comp_mod.F90

```diff
@@ -635,8 +637,6 @@ subroutine cime_pre_init1()
          iam=comp_comm_iam(it))
     if (iamroot_CPLID) output_perf = .true.
 
-    if (iamin_CPLID) complist = trim(complist)//' cpl'
-
```

Comment:
This version of complist is not properly initialized and was never used (see initialization in cime_init starting at line 1214).